src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+import { getUserById } from '../utils/GetUserByld';
```

Comment:
There's a typo in the import path for `getUserById`. Please ensure the file name is correctly spelled as `GetUserById` instead of `GetUserByld`.